node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
> Also, why is there so many ways to write a value? high, low, on (which also conflicts with being an EventEmitter), off, toggle, write

When I refactored the API for simplification, I wanted to encourage end developers (via new T2 docs) to migrate to "on", "off" and "toggle". The others exist for back compat with Tessel 1. 

> How about just having .value and .update(newValue, cb)? 

Why should the end developer be concerned with the "value"? They shouldn't need to care—the LED can only only be "on" or "off"

> Or do we need to maintain backwards compatibility?

That's why "high", "low" and "write" remain.